modules/k3s/master/instances_server.tf

```diff
@@ -87,9 +87,10 @@ resource "aws_instance" "master" {
     destination = "/tmp/ingresspolicy.yaml"
   }
   provisioner "remote-exec" {
-    inline = [
-      "chmod +x /tmp/k3s_master.sh",
-      "sudo /tmp/k3s_master.sh ${var.node_os} ${var.create_lb ? aws_route53_record.aws_route53[0].fqdn : "fake.fqdn.value"} ${var.install_mode} ${var.k3s_version} ${var.datastore_type} ${self.public_ip} \"${data.template_file.test.rendered}\" \"${var.server_flags}\"  ${var.username} ${var.password} ${var.k3s_channel}",
+    inline = [ <<-EOT
```

Comment:
Were you able to test by providing server/worker flags? I am not able to use this format in the dualstack setup

Reply:
i am yeah, actually this felt way better bc dont need to parse all "/ "

Reply:
Ok I will try out again and see

Reply:
Cool that works, thanks! 👍